MinaYonan123/mmwave-LENA-oran
# New run flags, parallel KPI logging to E2 term and txt files

1. Use E2 term and E2 log when --E2andLogging=bool
2. Set mmwave KPM E2 function using --KPM_E2functionID=double
3. Set lte KPM E2 function using --KPM_E2functionID=double
4. Set lte RC E2 function using --RC_E2functionID=double
5. New scenario for testing functions for parallel logging "scenario-zero-with_parallel_loging.cc" 
6. Added UE association with cell to scenario-zero-with_parallel_loging.cc
7. Added UE type differentiation to scenario-zero-with_parallel_loging.cc
8. Added Copyrights

To test E2 logging only
$ ./waf --run "scratch/scenario-zero-with_parallel_loging.cc --KPM_E2functionID=200 --RC_E2functionID=300 --e2TermIp=10.244.0.70 --simTime=100" 
To test file loging
$ ./waf --run "scratch/scenario-zero-with_parallel_loging.cc --simTime=100 --enableE2FileLogging=1"
To test parallel loging
$ ./waf --run "scratch/scenario-zero-with_parallel_loging.cc --KPM_E2functionID=200 --RC_E2functionID=300 --e2TermIp=10.244.0.70 --simTime=100 --E2andLogging=1"

Scenario should be merge with scenario-zero when code will be approved.

Also, code for Cell on/off is implemented, but scenario is not added to this version due to bugs.
